### PR TITLE
implement ORStackViewController

### PR DESCRIPTION
**Pull request by garnett**
Implements https://github.com/orta/ORStackView/issues/9

### CODE REVIEW

File: Classes/ios/ORStackViewController.m [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+- (ORStackScrollView *)scrollView {
+    return (ORStackScrollView *)self.view;
+}
+
+- (ORStackView *)stackView {
+    return self.scrollView.stackView;
+}
+
```
Comment (orta) [inline Classes/ios/ORStackViewController.m]: could this automate some of the top layout guide?

**Comment by orta** [comment]
Awesomeeeeee

File: Classes/ios/ORStackViewController.m (hunk 1 above)
Comment (garnett) [inline Classes/ios/ORStackViewController.m]: I am not sure how it should be handled inside ```ORStackViewController```, 
for example, for handling bottom bar in this PR I've used
```self.edgesForExtendedLayout = UIRectEdgeNone;``` 
inside sibling view controller and I thought it was correct approach of doing this. (at least I've did this for ```UITableViewController``` and it worked as expected)

Comment (orta) [inline Classes/ios/ORStackViewController.m]: Alright, this makes sense.  :dancers:

**Comment by orta** [comment]
Looks like the fail is legit on CI ( it's a snapshot on the ORSecondViewController ) any chance you can take a quick look? I'm on a computer all day, one this is in I'll release 3.0.0 :+1:

**Comment by garnett** [comment]
@orta sure I will

**Comment by garnett** [comment]
As it appears test never lie :smile: Now controller handles layout guides correctly and tests should pass.

**Comment by orta** [comment]
aceeeeeee

**Pull request merged**